libexec/scoop-shim.ps1

```diff
@@ -0,0 +1,241 @@
+# Usage: scoop shim <subcommand> [<shim_names>] [<command_path> [<args>...]] [-g(lobal)]
+# Summary: Manipulate Scoop shims
+# Help: Manipulate Scoop shims: add, remove/rm, list, info, alter, etc.
+#
+# To add a costom shim, use the 'add' subcommand:
+#
+#     scoop shim add <shim_name> <command_path> [<args>...]
+#
+# To remove a shim, use the 'remove' or 'rm' subcommand (CAUTION: this could remove shims added by app manifest):
```

Comment:
```suggestion
# To remove a shim, use the 'rm' subcommand (CAUTION: this could remove shims added by an app manifest):
```

For consistency with other scoop commands